buildtest/cli/report.py

```diff
@@ -576,14 +609,11 @@ def print_report(
         join_list = []
         title = title or f"Report File: {self.reportfile()}"
         table = Table(title=title, show_lines=True, expand=True)
-        consoleColor = checkColor(color=color)
-
         for field in self.display_table.keys():
             table.add_column(
-                f"[blue]{field}", overflow="fold", style=consoleColor
-            )  # change [blue] to console color if we want the entire console output to be uniform.
+                f"[{consoleColor}]{field}", overflow="fold", style=consoleColor
```

Comment:
now that i think about this i think the `style=consoleColor` will take care of coloring the field. See https://rich.readthedocs.io/en/stable/reference/table.html#rich.table.Table.add_column.

That being said i think the `f"[{consoleColor}]{field}"` could be changed to `table.add_column("field",overflow="fold", style=consoleColor)`.

Please try this out and if it works then please make the change otherwise leave it as is.

Reply:
So the only difference between those two is that with the original line(f"[{consoleColor}]{field}") the feild title text is also changed to the color. With "table.add_column("field",overflow="fold", style=consoleColor)" the feild name is in the default color tell me if thats the way you want it.

Reply:
Yeah let's only change the rowa that being said I think the original code is making everything be blue so let's keep this as is. I think using `style=consoleColor`  means we don't need to color inside the fitting.

Just make sure the column heading are blue and only the table rows are changed based on --color option

Reply:
You can do table.add_column(field, ..., style=consoleColor)

Reply:
done